nationwide_gh_config.yaml

```diff
@@ -0,0 +1,41 @@
+# Configuration file for standard GraphHopper routers.
+# 1 profile loaded with default speeds is defined for each vehicle type
+
+graphhopper:
+  datareader.file:
+  gtfs.file:
+  graph.location: transit_data/graphhopper
+  routing.ch.disabling_allowed: true
+  routing.max_visited_nodes: 1500000
+  graph.flag_encoders: car,bike,foot,truck
+  prepare.ch.threads: 16
+
+  # Profiles specifying vehicle and weightings for each mode type.
+  # For nationwide export, save time by not building contraction hierarchies
```

Comment:
This is the main important detail of why we need a separate graphhopper config for nationwide builds; we don't declare any contraction hierarchy profiles in this one, which means graphhopper doesn't build CHs (we don't need them because we're not routing nationwide yet), and the build becomes much faster